allennlp/tests/data/dataset_readers/atis_test.py

```diff
@@ -53,22 +38,6 @@ def test_atis_read_from_file(self):
         # (num_entities, num_utterance_tokens). We have two types
         # of entities: strings and numbers.
         assert world.linking_scores.shape[0] == \
-                len(world.valid_actions['string'] +
-                    world.valid_actions['number'])
+                len(world.valid_actions['number'])
         assert world.linking_scores.shape[1] == \
                 len(instance.fields['utterance'].tokens)
-
-        expected_linking_scores = [[0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1], # "westchester county" ->
```

Comment:
Why are you removing this?  Ah, I see, you're switching to learning an embedding for all of the string actions, instead of linking them.  I'm guessing that we'll still want to link these actions, instead of learning an embedding for them, but we can try it this way and see what happens.